src/main/java/com/palantir/gerrit/gerritci/ui/client/ProjectSettingsScreen.java

```diff
@@ -327,6 +430,10 @@ public void onClick(ClickEvent event) {
                         params.put("publishBranchRegex", makeXMLFriendly(publishBranchRegex.getText()));
                         params.put("publishCommand", makeXMLFriendly(publishCommand.getText()));
 
+                        params.put("cronJobEnabled", cronJobEnabled.getValue());
+                        params.put("cronCommand", makeXMLFriendly(cronCommand.getText()));
+                        params.put("cronJob", makeXMLFriendly(cronJob.getText()));
+
```

Comment:
From my understanding, all the branches that have been changed will build based on a timed schedule, unless specified otherwise.